.github/workflows/automatic-updates.yml

```
@@ -0,0 +1,37 @@
+name: Automatically update Docker image versions
+
+on:
+  schedule:
+    - cron: "15 * * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'iasuhiu3hdiuwdhiuwhqfs'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
```

```suggestion
      - uses: actions/setup-node@v3
```

although that should probably happen on main to all workflows

---

OK. I can make that change in this PR if you want.